library/me_calculator.inc.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/*
+* Morphine Calculator
+* @package OpenEMR
+* @link http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+*/
+
+
+
+      /*
+      *  This section is to save the ME total 
+      *
+      *
+      */
+    function saveMEinfo($pid, $enc, $sum)
+    {
+        if(empty($enc)){
+            return "<font color='red'>Please select an encounter to save this data</font>";
```

Comment:
translate and escape the constant using xlt
(note since you are including html in this function output, you need to translate it and html escape it here)

Reply:
I don't know how to escape the HTML

Reply:
``````
return "<font color='red'>" . xlt("Please select an encounter to save this data") . "</font>";
``````